jdbc/pom.xml

```
@@ -302,6 +295,8 @@
                       <exclude>org.apache.hadoop:hadoop-annotations</exclude>
                       <exclude>org.apache.hadoop:hadoop-auth</exclude>
                       <exclude>org.apache.hbase*:*</exclude>
+                      <exclude>org.apache.hive:hive-classification</exclude>  
+                      <exclude>org.apache.hive:hive-storage-api</exclude>   
```

These have extra trailing whitespace.

```suggestion
                      <exclude>org.apache.hive:hive-classification</exclude>
                      <exclude>org.apache.hive:hive-storage-api</exclude>
```